defguard-proxy.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=defguard proxy service
+Documentation=https://defguard.gitbook.io/defguard/
+Wants=network-online.target
+After=network-online.target
+
+[Service]
```

Comment:
@wojcik91 I think we can PIN proxy to a "user" that is not root. We can choose some UID & GID from system !=0